TESTING.md

```diff
@@ -79,6 +79,49 @@ The detected frameworks would be:
 
 If no `frameworks` folder exists, the current folder is assumed to be the root of a single framework. Note that the name of the framework displayed in this case (and above) is purely cosmetic, and the package name for the framework is determined from the catalog definition files.
 
+#### Running tests interactively
+
+The default when invoking `test.sh` is to run all tests defined for a given framework. It may, however be desirable to run a subset of tests interactively while developing a feature or the related tests themselves.
+
+The following command:
+```bash
+CLUSTER_URL=$(dcos config show core.dcos_url) ./test.sh -i
+```
+starts an interactive shell in a Docker container with the requirements for running the tests installed. The folder from which `./test.sh` was started is mounted into the container as `/build` and the default (or specified) SSH key is mounted at `/ssh/key`.
+
+**Note:** In order to run tests against a strict mode cluster and additional `-s` flag is required when starting the container.
+
+##### Setting up
+Before running test we perform the following steps inside the container:
+* Connect to the cluster: `dcos cluster setup ${CLUSTER_URL}`
+* Add the SSH key to the SSH agent: `eval $(ssh-agent -s); ssh-add /ssh/key`
+* Set the `PYTHONPATH` environment variable: `export PYTHONPATH=testing`
```

Comment:
Um, a relative `PYTHONPATH` entry seems wrong...

Reply:
Also, I cannot see where it's being set in the script?

Reply:
It's not set in the script, but is set in `__init__.py` and in `test_runner.sh`, but yes, this could be `/build/testing` in this case.